easybuild/easyconfigs/i/I-TASSER/I-TASSER-4.0.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = "PackedBinary"
+
+name = "I-TASSER"
+version = '4.0'
+
+homepage = 'http://zhanglab.ccmb.med.umich.edu/I-TASSER/'
+description = """I-TASSER is a set of pre-compiled binaries and scripts for protein structure and function
+modelling and comparison."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# Can't download from the web site automatically as registration is required.
+# The URL is, nevertheless, included here for reference.
+#source_urls = ['http://zhanglab.ccmb.med.umich.edu/cgi-bin/download_I-TASSER.cgi?ID=358515010=I-TASSER%(version)s.tar.bz2']
+sources = ['%(name)s%(version)s.tar.bz2']
+
+dependencies = [('BLAST', '2.2.26', "-Linux_x86_64"), ('Java', '1.7.0_51')] 
+
+postinstallcmds = ['/bin/chmod +x %(installdir)s/submit%(name)s']
```

Comment:
@valtandor just use `chmod` rather than `/bin/chmod`? Also, use `a+x` to make it more explicit?